doc/thingpedia-tutorial-hello-world.md

```diff
@@ -0,0 +1,80 @@
+# Tutorial 0: Hello World
+
+Welcome to Almond! This tutorial will get you started making your very first Almond device. 
+
+## Step 1: set up your device
+Go to the [Device Creation Page](/thingpedia/upload/create), fill in the following basic information 
+about the device:
+
+- ID: `hello.<your-name>` (Each device in Thingpedia needs an unique ID. Make sure your ID hasn't been used!)
```

Comment:
We should encourage people to follow reverse DNS notation (that is, use a domain they control, such as their github page), with the exception that they can use their own username as the first component.

Reply:
Okay so `<your-name>.hello`?